packages/manager/apps/public-cloud/package.json

```diff
@@ -67,6 +67,7 @@
   },
   "devDependencies": {
     "@ovh-ux/manager-webpack-config": "^3.0.6",
+    "webpack": "^4.30.0",
```

Comment:
Isn't this brought with `manager-webpack-config` ? https://github.com/ovh-ux/manager-webpack-config/blob/master/package.json#L60